fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_objectives.dm

```diff
@@ -208,8 +208,48 @@
 
 /// Convert a crewmate
 /datum/objective/bloodsucker/embrace
+	name = "embrace"
+
+// EXPLANATION
+/datum/objective/bloodsucker/embrace/update_explanation_text()
+	. = ..()
+	explanation_text = "Use the Candelabrum to Rank your Favorite Vassal up enough to become a Bloodsucker."
+
+// WIN CONDITIONS?
+/datum/objective/bloodsucker/embrace/check_completion()
+	var/datum/antagonist/bloodsucker/bloodsuckerdatum = owner.current.mind.has_antag_datum(/datum/antagonist/bloodsucker)
+	if(!bloodsuckerdatum || bloodsuckerdatum.my_clan != CLAN_VENTRUE)
+		return FALSE
+	for(var/datum/antagonist/vassal/vassaldatum in bloodsuckerdatum.vassals)
+		if(vassaldatum.owner && vassaldatum.favorite_vassal)
+			if(vassaldatum.owner.has_antag_datum(/datum/antagonist/bloodsucker))
+				return TRUE
+	return FALSE
+
+//////////////////////////////////////////////////////////////////////////////////////
+
+/// Enter Frenzy repeatedly
+/datum/objective/bloodsucker/frenzy
+	name = "frenzy"
+
+/datum/objective/bloodsucker/frenzy/New()
+	target_amount = rand(3,4)
```

Comment:
Why isn't this in generate_objective?

Reply:
And now that Im looking at it... a lot of bloodsucker objectives override generate_objective with the same exact content as the parent objective (Which is just calling update_explanation_text())